pkg/serverless/trace/propagation/extractor.go

```diff
@@ -45,9 +45,10 @@ type Extractor struct {
 
 // TraceContext stores the propagated trace context values.
 type TraceContext struct {
-	TraceID          uint64
-	ParentID         uint64
-	SamplingPriority sampler.SamplingPriority
+	TraceID           uint64
+	TraceIDUpper64Hex string
```

Comment:
So, Java, .NET, and Go tracers will automatically recognize this new attribute in the trace context, and add it to the span metadata?

Reply:
That's a good question. My understanding is the extension starts the trace and adds the TraceID+ Upper 64. The Java, .Net, or Go tracer creates spans and then sends them to the extension. The extension adds them to the trace it created. I'll need to verify that the extension is doing enough with respect to the Upper 64 bits (i.e. do they need to be added to the _meta._dd.p.tid field for every span? Or just the root span?)